src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java

```diff
@@ -219,7 +156,17 @@ private DslPart createResponseForGetTaskWithWarnings() throws JsonProcessingExce
                                 .stringType("warningText", "Text1")
                             )
                         )
-                        .stringType("case_management_category", "Some Case Management Category")
```

Comment:
workType mapping is no longer here? -> .stringType("work_type_id", "hearing_work")

Reply:
solved thanks